### ISSUE TITLE
Get output masks only on desired classes. 

### ISSUE THREAD

**Issue by HamzaJavaid-gh** [facebookresearch/Mask2Former#48]
Hi, 
Is there a way to get masks in the output of  only 1 or 2 specified classes from ADE20k or COCO?

**Comment by bowenc0221** [comment]
No supported yet but you can modify the inference code to only return masks you want.

**Comment by Saafke** [comment]
This sample code may help you.

```
# Run the model on the image
predictions, visualized_output = demo.run_on_image(img)

# Get only the person mask (binary)
panoptic_img = predictions['panoptic_seg'][0].detach().numpy()
person_id = 1
person_img = np.where(panoptic_img == person_id, 1, 0) * 255
person_img = person_img.astype(np.uint8)
```
